src/server_manager/web_app/app.ts

```diff
@@ -334,7 +336,8 @@ export class App {
       this.digitalOceanAccount = digitalOceanAccount;
       this.appRoot.digitalOceanAccount = {
         id: this.digitalOceanAccount.getId(),
-        name: await this.digitalOceanAccount.getName()
+        name: await this.digitalOceanAccount.getName(),
+        cloudId: DisplayCloudId.DO,
```

Comment:
The DO account object lives in a DO-specific variable. A cloudId doesn't belong there, since its cloud is clear.

Use DisplayCloudId directly in app-root's `_getCloudId`

Reply:
Done.  (Note that we'll need something like this for multi-account support.)

Reply:
It depends on the multi-cloud implementation. For instance, you can still have a per-cloud account list